.travis.yml

```diff
@@ -1,34 +1,33 @@
-# See https://github.com/silverstripe/silverstripe-travis-support for setup details
 language: php
 
-sudo: false
-
 env:
   global:
-    - COMPOSER_ROOT_VERSION=1.0.x-dev
-    - CORE_RELEASE=4
-    # The path of the module when installed by composer
-    - MODULE_PATH=iframe
+    - COMPOSER_ROOT_VERSION=5.0.x-dev
```

Comment:
This should match whatever the module we're in is - in this case probably 2.0.x-dev